package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "marvin",
+  "version": "0.0.0",
+  "description": "An API driven skin for MediaWiki",
+  "scripts": {
+    "start": "run-p -n dev:*",
+    "dev:client": "NODE_ENV=development webpack -w",
+    "dev:server": "NODE_ENV=development nodemon -w dist/client -w src/server/ src/server/index.js",
+    "prod:build": "NODE_ENV=production webpack -p",
+    "format": "prettier --write 'src/**/*.js'",
+    "lint": "eslint 'src/**/*.js'",
```

Comment:
Great advice, thanks! I'm submitting the changes as new commits on the same PR to not deal with rebasing and force pushing losing context for the comments.

* ad7e5b2 - changing the lint script to adopt the lint:all advice
* eb87366 - add --cache to the lint script & add .eslintcache to gitignore
* 8128fd1 - add --max-warnings 0 to lint script

Re: globs, Most programs understand string globs based on the package [node-glob](https://github.com/isaacs/node-glob) so it is better to pass a string than to rely on the shell's globbing (for example npm uses sh to run the scripts, which doesn't have proper globstar support).

Regarding `--ext .js,.json,.html` it seems overkill to set up linting for them given we don't have any files of that kind, feel free to submit a followup commit, I'd be happy to merge.

Re: Replace 'src/**/*.js', I believe that having the explicit path to the JS files there kind of makes it easy to check the paths of the files for other tools like prettier. Happy to re-evaluate later if we increase the amount of eslint config